fortanix-vme/fortanix-vme-runner/src/lib.rs

```diff
@@ -413,11 +417,34 @@ impl Server {
         }
     }
 
+    fn handle_request_close(self: Arc<Self>, enclave_port: u32, enclave: &mut VsockStream) -> Result<(), IoError> {
+        let cid: u32 = enclave.peer().unwrap().parse().unwrap_or(vsock::VMADDR_CID_HYPERVISOR);
+        let addr = VsockAddr::new(cid, enclave_port);
+        if let Some(listener) = self.remove_listener(&addr) {
+            // Close `TcpListener`
+            drop(listener);
+        } else {
+            // Info not found, possibly not a listener socket
```

Comment:
This is quite unexpected, right? Perhaps it would be good to print a warning here.

Reply:
It'd be better to return an error message here. But returning errors is a bit of a mess at the moment. I created a separate ticket for it. Would you accept a `println` for now, and improve things when errors and logging are added?